assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -193,22 +199,29 @@ export default function SetupMainPAX( { finishSetup } ) {
 		setShowPaxAppQueryParam( PAX_SETUP_STEP.LAUNCH );
 	}, [ navigateTo, setShowPaxAppQueryParam, hasAdwordsScope, oAuthURL ] );
 
-	const onLaunch = useCallback( ( app ) => {
-		paxAppRef.current = app;
-	}, [] );
+	const onLaunch = useCallback(
+		async ( app ) => {
+			await trackEvent( `${ viewContext }_pax`, 'pax_launch' );
```

Comment:
No need to await here.

Reply:
@aaemnnosttv this has been resolved in my latest set of commits.